docs/source/how-to/deploy-dask-clusters.rst

```diff
@@ -16,6 +31,9 @@ Dask has two families of task schedulers:
     more features, but also requires a bit more effort to set up.  It can
     run locally or distributed across a cluster.
 
+.. image:: ../images/distributed-overview.svg
```

Comment:
This needs a sentence that makes it clear that distributed scheduler == distributed cluster.

Reply:
Thanks! I added this to the caption, let me know what you think!